.github/workflows/ci-crossdock.yml

```diff
@@ -45,12 +45,18 @@ jobs:
 
     - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
 
-    - name: Build, test, and publish crossdock image
+    - name: Build crossdock image (main branch only)
+      if: github.ref_name == 'main'
```

Comment:
we don't need these ifs, unlike main binaries, crossdock always can be built on one architecture only

Reply:
Yes, I will be removing that.
The main change of this commit was only the Makefile